src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.PackageDependencyResolution.targets

```diff
@@ -20,7 +20,7 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   <!-- General Properties -->
   <PropertyGroup>
-    <ProjectAssetsFile Condition="'$(ProjectAssetsFile)' == ''">$(MSBuildProjectExtensionsPath)/project.assets.json</ProjectAssetsFile>
+    <ProjectAssetsFile Condition="'$(ProjectAssetsFile)' == ''">$(MSBuildProjectExtensionsPath)project.assets.json</ProjectAssetsFile>
```

Comment:
This change doesn't help us anyway, right?
Should we revert it and make in the build incrementalism tracking issue?